src/seedu/addressbook/AddressBook.java

```diff
@@ -207,14 +206,52 @@
      */
 
     public static void main(String[] args) {
-        showWelcomeMessage();
-        processProgramArgs(args);
-        loadDataFromStorage();
+
+        showToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
```

Comment:
The changes in main() seems intended for A3?

Remember to start a fresh branch from master for each activities in future. :-)